Loop/Managers/LoopDataManager.swift

```diff
@@ -184,12 +199,15 @@ class LoopDataManager {
 
      - parameter resultsHandler: A closure called once the values have been retrieved. The closure takes the following arguments:
         - predictedGlucose:     The calculated timeline of predicted glucose values
+        - insulinEffect:        The predicted effect of insulin
+        - carbEffect:           The predicted effect of carbohydrates
```

Comment:
I'm going to soon; I just need to figure out how to turn them into BG lines.  carbEffect -> predicted BG if only carbs were considered, and insulinEffect -> predicted BG if only insulin were considered.

But I can remove them for this commit, if you'd like.